core/src/main/java/com/airwallex/airskiff/flink/udx/StdDevFunction.java

```diff
@@ -0,0 +1,40 @@
+package com.airwallex.airskiff.flink.udx;
+
+import org.apache.flink.table.api.dataview.ListView;
+import org.apache.flink.table.functions.AggregateFunction;
+
+import java.util.List;
+
+public class StdDevFunction extends AggregateFunction<Double, StdDevAccumulator> {
+  @Override
+  public Double getValue(StdDevAccumulator acc) {
+    List<Double> list = acc.nums.getList();
+    if(list.size() == 0 || list.size() == 1) {
+      return null;
+    } else {
+      Double avg = list.stream().mapToDouble(a -> a).average().orElse(0.0);
```

Comment:
can skip this traversal if we have avg in the accumulator.

Reply:
got it